CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # nf-core/bamtofastq: Changelog
 
+## v1.2.0 - Marie Curie
```

Comment:
```suggestion
## v1.2.0 - Anna Winlock
```

since I was going for Woman in computer science, Marie Curie - while of course important and famous - was not a computer scientist